class_student.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(id, age, classroom, name: 'Unknown', parent_permission: true)
+    super(id, age, name, parent_permission)
```

Comment:
- [ ] For each person that would be created, a random id will be generated as already mentioned above hence we won't need id parameter when creating a student :+1: Let's kindly get rid of it here.

Reply:
Let's focus on the requirements bro